app/components/profile/picture_component.html.erb

```diff
@@ -1,6 +1,10 @@
 <figure class="rounded-full w-64 h-64 mx-auto sm:mx-0 relative border-2 border-black bg-black mb-2">
-  <%= image_tag profile_picture_src, class: 'w-full object-cover rounded-full' %>
-  <%= tag.figcaption(class: "absolute bottom-0 translate-y-1 z-90 py-2 rounded-lg w-full text-center font-bold #{figcaption_colors}") do %>
-    <%= profile.job_search_status.humanize.titleize %>
-  <% end %>
+  <div class="relative inline-block w-full object-cover rounded-full">
+    <%= image_tag profile_picture_src, class: 'h-auto max-w-full block' %>
+    <% if open_to_work? %>
+      <div class="absolute top-0 left-0 w-full">
+        <%= render inline: Rails.root.join('app/assets/images/open_to_work.svg').read %>
```

Comment:
Dunno whether this is the most idiomatic way to render the svg inline, but seems like it works. Maybe could be pushed to the component, just wanted to get something up and running

Reply:
I've most often seen people use [this gem](https://github.com/jamesmartin/inline_svg), but I'm not immediately opposed to this if it works well for this  one-off usecase. Not sure how many svgs we'll be using.